pokemongo_bot/event_handlers/telegram_handler.py

```diff
@@ -359,4 +437,4 @@ def handle_event(self, event, sender, level, formatted_msg, data):
                     return
                     
             if self.tbot is not None: # tbot should be running, but just in case it hasn't started yet
-                self.chat_handler.sendMessage(chat_id=uid, parse_mode='Markdown', text=msg)
+                self.chat_handler.sendTeleMessage(chat_id=uid, parse_mode='Markdown', text=msg)
```

Comment:
not supposed to be here - "sendTeleMessage" must be in the telegram_handler, not in the generic chat_handler!